program_admin/instructions.py

```diff
@@ -14,6 +14,7 @@
 COMMAND_ADD_PRICE = 4
 COMMAND_ADD_PUBLISHER = 5
 COMMAND_DEL_PUBLISHER = 6
+COMMAND_INIT_PRICE = 4
```

Comment:
This needs to be 9

Reply:
I even put it in the right order. I think I hit undo by accident.